app/lib/actions/katello/pulp_selector.rb

```diff
@@ -2,12 +2,31 @@ module Actions
   module Katello
     class PulpSelector < Actions::Base
       def plan(backend_actions, repository, smart_proxy, *args)
+        pass_only_args = args.dig(0, :args_only) || false
```

Comment:
Is this still used?

Reply:
Removed pass_only_args..No longer needed..